s5-hooks/src/components/PostView/postView.js

```diff
@@ -1,11 +1,18 @@
-import React from 'react'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
+import Comments from '../Comments/comments'
 
 export default function PostView({id, title, body, userId}) {
+  const [showComments, setShowComments] = useState('false')
+ 
+  let emptyArray = []
+
   return (
     <div style={{textAlign: "left"}}>
         <h4>Title: {title}</h4>
         <p>Content: {body}</p>
+        <button onClick={() => {setShowComments(!showComments)}}>Comments</button>
+        {(!showComments) ? <Comments  id={id}/> : emptyArray }
```

Comment:
In loc de empty array poti pune `null`

Reply:
Poti chiar optimiza acest statement in felul asta:
`{showComments && <Comments id={id} />}`